static/js/index.js

```diff
@@ -3592,6 +3592,19 @@ function getWeatherMarker (weather, ts) {
 }
 
 function getNestMarker (nest, geojson, ts) {
+    const pkmn = masterfile.pokemon[nest.pokemon_id];
+    let typesIcon = '';
+    if (pkmn) {
+        const types = pkmn.types;
+        if (types && types.length > 0) {
+            //if (types.length === 2) {
+                typesIcon += '<span class="text-nowrap"><img src="/img/nest/nest-' + types[0].toLowerCase() + '.png" height="56" width="auto" style="-webkit-mask-image: -webkit-gradient(linear, left 90%, left bottom, from(rgba(0,0,0,1)), to(rgba(0,0,0,0)))"></span>&nbsp;';
```

Comment:
```suggestion
                typesIcon += '<span class="text-nowrap"><img src="/img/nest/nest-' + types[0].toLowerCase() + '.png" height="45" width="auto" style="-webkit-mask-image: -webkit-gradient(linear, left 90%, left bottom, from(rgba(0,0,0,1)), to(rgba(0,0,0,0)))"></span>&nbsp;';
```
I found this size to be a little more appropriate but dealer's choice.